src/test/nodestore/DatabaseShard_test.cpp

```diff
@@ -1273,6 +1287,7 @@ class DatabaseShard_test : public TestBase
                     {
                         // A rotation occurred during shard import. Not
                         // necessarily an error
+
```

Comment:
Nit: sequence can be renamed to ledgerSeq

Reply:
Done